SETUP_ARCH.md

```diff
@@ -0,0 +1,23 @@
+# Snowdrift setup instructions for Arch Linux
+
+Arch has the benefit of being very well-documented, so this document is
+very short!
+
+1.  [Install Haskell][1]. While you are at it, make sure you have the
+    `base-devel` group installed.
+2.  [Install PostgreSQL][2]. You don't need to go through setting up a
+    user or anything like that. Snowdrift will do that for you.
```

Comment:
Hmm. You should look at the link I was referencing. It has "installing" at the top, and all sorts of other junk below that. The head of that junk is "setting up a user", which is why I said "setting up a user or anything like that".